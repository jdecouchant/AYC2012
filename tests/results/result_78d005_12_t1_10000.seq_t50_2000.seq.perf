
Elapsed real time                  0:00.09
CPU-seconds spent in kernel        0.02
CPU-seconds spent in user          0.32
Percentage of the CPU got          359%
Maximum resident set size          4512 Kbytes.
Average resident set size          0 Kbytes.
data+stack+text memory use         0 Kbytes.
Average size unshared data area    0 Kbytes.
Average unshared stack space       0 Kbytes.
Average size shared text space     0 Kbytes.
Number of major page faults        0
Number of recoverable page faults  919
Number of context-switch           93
Number of waits                    69
Number of file system inputs       0
Number of file system outputs      8
