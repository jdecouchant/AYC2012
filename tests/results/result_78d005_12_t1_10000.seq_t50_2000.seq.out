Sequence N°0
4129 4141 311 323
Sequence N°1
5095 5106 755 766
7072 7083 1170 1181
Sequence N°10
Sequence N°11
Sequence N°12
8127 8138 1842 1853
Sequence N°13
9804 9815 470 481
Sequence N°14
565 576 364 375
Sequence N°15
3285 3298 925 938
Sequence N°16
5709 5720 1265 1276
Sequence N°17
9540 9551 663 674
Sequence N°18
Sequence N°19
Sequence N°2
4284 4295 1889 1900
Sequence N°20
7769 7781 1535 1547
Sequence N°21
Sequence N°22
Sequence N°23
Sequence N°24
6160 6171 296 307
9601 9612 1817 1828
Sequence N°25
2717 2729 854 866
Sequence N°26
8029 8041 1100 1112
Sequence N°27
Sequence N°28
1924 1935 480 491
Sequence N°29
Sequence N°3
6900 6912 1310 1322
Sequence N°30
5121 5132 85 96
Sequence N°31
7389 7400 1421 1432
Sequence N°32
8487 8498 1171 1182
Sequence N°33
4304 4315 1892 1903
Sequence N°34
1163 1174 1808 1819
Sequence N°35
2084 2095 286 297
8959 8971 16 28
Sequence N°36
1266 1277 265 276
Sequence N°37
Sequence N°38
4316 4327 197 208
4727 4738 847 858
Sequence N°39
Sequence N°4
5967 5978 324 335
Sequence N°40
3065 3076 452 463
Sequence N°41
7429 7440 1334 1345
Sequence N°42
Sequence N°43
5774 5785 910 921
9658 9670 1231 1243
Sequence N°44
Sequence N°45
2368 2380 669 681
Sequence N°46
Sequence N°47
Sequence N°48
6582 6594 1718 1730
Sequence N°49
20 31 1172 1183
4190 4201 1262 1273
6687 6698 1 12
Sequence N°5
Sequence N°6
Sequence N°7
5170 5181 121 132
Sequence N°8
Sequence N°9
